students/jeremy_m/lesson05_exercises/mailroom_part_3.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python3
+
+# lesson 03 Exercise - String Formatting Lab
+# Jeremy Monroe
+
+import os
+import pathlib
+import time
+
+donors = {'Charlize Theron': [134000],
+          'Charlie Boorman': [15, 5],
+          'James Franco': [25, 250, 2, 500],
+          'Nike': [22000],
+          'Count Chocula': [1257633, 2532790]
+          }
+
+# donor_totals = {}
+
+user_input = ''
+
+messages = {'start': ("What would you like to do?\n"
+                      "1: to Send a Thank You\n"
+                      "2: Create a Report\n"
+                      "3 to create letters for all donors\n"
+                      "quit: to Quit"
+                      ),
+            'thanks': ("Please input the donors name "),
+            'thank_you_message': ("{:^42}\n"
+                                  "{:^42}\n"
+                                  "For your incredibly generous donation of:\n"
+                                  "{:>19}{:<23,}\n\n"),
+            'donor_letter': ("{:^41}\n"
+                             "Thank you so much for your generous donation of:\n"
+                             "{:>21}{:,}\n"
+                             "We will always remember your money fondly."),
+            'letters_confirmation': 'Is this ok?\nType y or n:'
+            }
+
+
+def letters_for_all():
+    """ This function creates a directory in the cwd and fills that with an
+        individual file (letter) for each donor.
+    """
+
+    clear_screen()
+    print('Note: This operation will create a directory in the current working directory')
+    if get_user_input(messages['letters_confirmation']).lower() == 'y':
```

Comment:
this function `get_user_input` doesn't really buy you anything as a function as its still just one line to do the same thing:
```
if input(messages['letters_confirmation']).lower() == 'y':
```